tests/test-cases/image/.gitlab-ci.yml

```diff
@@ -5,3 +5,10 @@ test-job:
   image: ${PHP_IMAGE_URL}
   script:
     - echo "Test something"
+
+test-entrypoint:
+  image: firecow/gitlab-ci-local-test-image
+  script:
+    - pwd
+    - echo "Test Entrypoint"
+    - ls -all
```

Comment:
According to the test you have written you do not use the information from the ls command. Perhaps this should be removed?